src/igvfd/tests/test_upgrade_matrix_file.py

```diff
@@ -24,3 +24,12 @@ def test_matrix_file_upgrade_5_6(upgrader, matrix_file_v5):
     assert 'derived_from' not in value
     assert 'file_format_specifications' not in value
     assert value['schema_version'] == '6'
+
+
+def test_matrix_file_upgrade_6_7(upgrader, matrix_file_v6):
+    value = upgrader.upgrade('matrix_file', matrix_file_v6, current_version='6', target_version='7')
+    assert 'principal_dimension' not in value
```

Comment:
Possible copy paste error. I think you are wanting to assert that the old property name "dimension1" is not in value. (Same with the next line)